core/src/main/java/io/github/iprodigy/cache/core/provider/SimpleMapProvider.java

```diff
@@ -0,0 +1,20 @@
+package io.github.iprodigy.cache.core.provider;
+
+import io.github.iprodigy.cache.api.Cache;
+import io.github.iprodigy.cache.api.ICacheSpec;
+import io.github.iprodigy.cache.core.AbstractCacheProvider;
+import io.github.iprodigy.cache.core.delegate.GenericMapCacheDelegate;
+
+import java.util.HashMap;
+
+/**
+ * A very simple map cache provider for testing purposes
+ */
+public final class SimpleMapProvider extends AbstractCacheProvider {
```

Comment:
Is it possible to only have this defined in the test packages?

Reply:
It's only used for testing, so moving it to the test source sounds good